components/Tabs/examples/MDCTabBarViewTypicalExampleViewController.m

```diff
@@ -74,6 +74,9 @@ - (void)viewDidLoad {
   [super viewDidLoad];
   self.title = kExampleTitle;
 
+  UIScrollView *dummyScrollView = [[UIScrollView alloc] init];
+  [self.view addSubview:dummyScrollView];
```

Comment:
Please document that this is being added so that the app bar does not inject content insets into the tab bar. Please also pull this out to a separate PR.

Reply:
Doing in #7772